tests/test_docker.py

```diff
@@ -25,6 +25,20 @@ def test_volume_activate(agent, responses):
 
 
 @if_docker
+def test_volume_activate_driver(agent, responses):
```

Comment:
You should have a test for activate, deactivate, and remove.  Make sure the case where the volumes are not for a driver are covered.  They may be already, not totally sure.  Just look at the existing tests.